### PR TITLE
Notify about chat mentions in editor

### PR DESCRIPTION
**Pull request by 12pm**
Not sure if this is a good way to display this

image

### CODE REVIEW

File: src/game/editor/editor.cpp [hunk 1]
```diff
@@ -5678,6 +5678,24 @@ void CEditor::Render()
 		StatusBar.Margin(2.0f, &StatusBar);
 	}
 
+	// show mentions
+	if(m_GuiActive && m_Mentions)
+	{
+		char aBuf[16];
+		if(m_Mentions == 1 || m_Mentions > 9)
```
Comment (def-) [inline src/game/editor/editor.cpp]: please make 2 separate if conditions out of this

File: src/game/editor/editor.cpp [hunk 2]
```diff
@@ -5678,6 +5678,24 @@ void CEditor::Render()
...
+		{
+			str_copy(aBuf, m_Mentions == 1 ? "1 New Mention" : "9+ New Mentions", sizeof(aBuf));
+		}
+		else
+		{
+			str_format(aBuf, sizeof(aBuf), "%d New Mentions", m_Mentions);
```
Comment (def-) [inline src/game/editor/editor.cpp]: I'd prefer "5 new mentions" since it's not a title

**Comment by def-** [comment]
Generally it doesn't look bad. We already have notification sounds in editor too, right?
I guess there is also some overlap with https://github.com/ddnet/ddnet/issues/1330 but that's probably not so useful when editing, so I'd be fine with this change.

File: src/game/client/components/chat.cpp [hunk 3]
```diff
@@ -13,6 +13,7 @@
 #include <game/generated/client_data.h>
 
 #include <game/client/gameclient.h>
+#include <game/editor/editor.h>
```
Comment (Ryozuki) [inline src/game/client/components/chat.cpp]: I don't think you need to add CEditor to the chat component, CComponent already has access to the interface IEditor which is enough for this right?

**Comment by 12pm** [comment]
Applied all suggestions

File: src/game/editor/editor.cpp [hunk 4]
```diff
@@ -5678,6 +5678,28 @@ void CEditor::Render()
 		StatusBar.Margin(2.0f, &StatusBar);
 	}
 
+	// show mentions
+	if(m_GuiActive && m_Mentions)
+	{
+		char aBuf[16];
+		if(m_Mentions == 1)
+		{
+			str_copy(aBuf, "1 new mention", sizeof(aBuf));
```
Comment (def-) [inline src/game/editor/editor.cpp]: Let me merge and also Localize them.

Comment (12pm) [reply src/game/editor/editor.cpp]: Oh didn’t know editor is localized

**Pull request merged**